README.md

```diff
@@ -1,16 +1,27 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2324S2-CS2103T-T13-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S2-CS2103T-T13-2/tp/actions)
 
 [![codecov](https://codecov.io/gh/AY2324S2-CS2103T-T13-2/tp/graph/badge.svg?token=X8FVKZE2OZ)](https://codecov.io/gh/AY2324S2-CS2103T-T13-2/tp)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+# InternBook
+
+As a student looking for internships, it is often hard to keep track of the many different internship applications.
+One of the solutions is to do so in an excel sheet, 
+but such a user interface can become very hard to manage when there are many concurrent applications going on.
+Thus, having a CLI based interface can make it much easier for data input and query.
+Our app hopes to reduce the effort needed to track internships, so that students can spend the time searching instead. 
+Our app will aim to record employers’ emails, job descriptions and application dates.
+
+## Getting Started
+1. Ensure that Java 11 is available on your machine.
+2. Download the jar file from here.
+3. Navigate to the file's location in the shell of your choice.
+4.Run the jar file with the follow command
```

Comment:
Everything looks good to me except for this i think there is a minor formatting issue, just need a space at after the fullstop